.github/scripts/generate_binary_build_matrix.py

```diff
@@ -84,6 +84,19 @@
         "nvidia-nvtx-cu12==12.6.77; platform_system == 'Linux' and platform_machine == 'x86_64' | "
         "nvidia-nvjitlink-cu12==12.6.77; platform_system == 'Linux' and platform_machine == 'x86_64'"
     ),
+    "xpu": (
```

Comment:
Hi @chuanqi129 please add these packages here: https://github.com/pytorch/test-infra/edit/main/s3_management/manage.py

Reply:
And perhaps here as well: https://github.com/pytorch/test-infra/blob/main/s3_management/update_dependencies.py#L11 
So we can update them via script.

Reply:
Thanks @atalman help to point it out, the PR has been submitted https://github.com/pytorch/test-infra/pull/5965. Please help to review it